### DISCUSSION TITLE
Failed enable Google Auth

### DISCUSSION THREAD

**Discussion by syaifulhuseinnn** [supabase/supabase#5878] [Questions]
Hi, I'm learning Supabase. I'm new to this stuff.

I'm going to enable Google Auth on my project but I always got this error when I save the config:

![image](https://user-images.githubusercontent.com/38772436/157573140-241db7be-9f37-4966-8c52-1c8b7a6b87aa.png)

I have followed the documentation: [https://supabase.com/docs/guides/auth/auth-google](https://supabase.com/docs/guides/auth/auth-google)
I have tried to delete the project and create a new project again but it doesn't work. Could someone help me? Maybe I missed something.

**Comment by GaryAustin1** [comment]
Found this:
https://github.com/supabase/supabase/issues/5327
That is on the same page you are on.
